modules/yaml_generator.py

```diff
@@ -56,18 +66,19 @@ def generate_yaml(self, conversation, string, **kwargs):
         response = llm_chain(inputs={"string": string})
 
         # https://stackoverflow.com/a/63082323/2328066
-        regex = r"(?:\n+|\A)?(?P<code_block>\`{3,}yaml\n+[\S\s]*\`{3,})"
```

Comment:
Assuming that we don't need any extra parsing on response. Looks good to me.

Reply:
this is another case of "eventually we will" (at a minimum we should be yaml linting the response to confirm it is indeed a valid yaml response and has no formatting around it) but for now this change gets the api endpoint working again w/ the current model